### PR TITLE
WDP190705-9 - Transition color and background  for :hover elements

### PR DESCRIPTION
**Pull request by EnDriash**
1. Resolved animation problem/
2. Added _general and mixinis&placeholders new .scss files/
3. Created sections directory and moved footer with header into it

### CODE REVIEW

**Comment by EnDriash** [comment]
Done

File: package.json [hunk 1]
```diff
@@ -35,15 +35,15 @@
     "eslint-plugin-standard": "^4.0.0",
     "husky": "^1.3.1",
     "lint-staged": "^8.1.0",
-    "parcel-bundler": "^1.11.0",
+    "parcel-bundler": "^1.12.3",
```
Comment (stepek) [inline package.json]: Why you update dependencies in this commit?

File: package.json [hunk 2]
```diff
@@ -35,15 +35,15 @@
...
     "posthtml-include": "^1.2.0",
     "prettier": "^1.15.3",
     "pretty-quick": "^1.8.0",
     "rimraf": "^2.6.3",
     "sass": "^1.15.3"
   },
   "dependencies": {
-    "jquery": "^3.3.1",
+    "jquery": "^3.4.1",
```
Comment (stepek) [inline package.json]: Why you update dependencies in this commit?

Comment (EnDriash) [reply package.json]: i think it can be accident. I resolved and commited this change.

Comment (stepek) [reply package.json]: avoid this in the future

**Comment by EnDriash** [comment]
Resolved conflicts. Ready to merge.
Data: 31.07.19r.

**Pull request merged**